style.css

```
@@ -1,456 +1,1102 @@
+  }
+
+  section.project-popup div.project-wrapper {
+    display: flex;
+    flex-direction: row;
+  }
+
+  section.project-popup div.project-wrapper div.main-project-image {
+    width: 65%;
+  }
+
+  section.project-popup div.project-wrapper div.project-wraper-p-btn {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 25px;
+    padding-bottom: 4px;
+    padding-top: 10px;
+  }
+
+  div.contact-main {
+    display: grid;
+    grid-template-columns: 1fr 2fr;
+    padding-top: 130px;
+    padding-bottom: 130px;
+  }
+
+  div.contact-main p {
+    text-align: left;
+  }
+
+  section#contact {
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  form.contact-form li.button-wraper {
+    margin-top: 24px;
+    justify-content: left;
+  }
+
+  form.contact-form button {
+    margin: 0;
+  }
+
+  div.line-aboutme {
+    padding-left: 141px;
+    padding-right: 141px;
+    margin-top: 100px;
+  }
+
+  div.contact-heading p {
+    padding-top: 0;
+  }
+
+  header.work-heading {
+    display: flex;
+    flex-direction: row;
+    align-items: start;
+    justify-content: start;
+    width: 120%;
+    margin: 0 114px 130px 0;
+  }
+
+  header.work-heading hr {
+    border: none;
+    height: 0.5px;
+    background: #212121;
+    gap: 24px;
+    margin-top: 12%;
+    width: 33%;
+  }
+
+  header.work-heading h2 {
+    text-align: left;
+  }
+
+  hr.work-line {
+    width: 27%;
+  }
+
+  section.work-section {
+    padding-left: 141px;
+    padding-right: 141px;
+  }
+
+  section.work-section article {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 20px;
+  }
+
+  section.work-section article div.main-project {
+    grid-area: 1/1/1/3;
+    flex-direction: row;
+    align-items: start;
+  }
+
+  section.work-section article div.main-project div {
+    width: 50%;
+  }
+
+  section.work-section article div.main-project div.main-project-details {
+    padding-left: 24px;
+    padding-right: 24px;
+  }
+
+  section.work-section article div.main-project div.main-project-details h3 {
+    margin-top: 0;
+  }
+
+  section.work-section article div.sub-projects {
+    width: 100%;
+  }
+
+  section.work-section article div.sub-projects div.sub-project-container {
+    height: 100%;
+    padding: 50px 16px;
+    display: flex;
+    flex-direction: column;
+    justify-content: end;
+  }
+
+  section.work-section article div.first {
+    border: 2px solid black;
+  }
+
+  section.work-section article div.first div.sub-project-container h3.title {
+    display: none;
+  }
+
+  section.headline-section div.headline p {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  section.work-section article div.first div.sub-project-container p {
+    display: none;
+  }
+
+  section.work-section article div.first div.sub-project-container div {
+    display: none;
+  }
```

Let's try to make our code dry and save some lines of code😉. as we will be obeying some best practices rule doing so😉.

```suggestion
  section.work-section article div.first div.sub-project-container div p h3.title {
    display: none;
  }
```

doing this, you also won't need the block of code from lines 780 to 782👍.